src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -202,25 +202,34 @@ export const ConsultationDetails = (props: any) => {
                   </div>
```

Comment:
i think green color with tick notation is best for showing `verified`. I found [this icon list](https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=verify).